common/tandemClient.js

```diff
@@ -189,6 +253,37 @@ export class TandemClient {
         return results;
     }
 
+    /**
+     * Returns stream elements from given model.
+     * @param {string} urn - URN of the model.
+     * @param {string[]} [columnFamilies] - optional list of columns
+     * @returns {Promise<object[]>}
+     */
+    async getStreams(urn, columnFamilies = [ ColumnFamilies.Standard ]) {
```

Comment:
Likely, for stream one might also be interested in fetching tandem native family as well.